_releases/2023-07-11-1.9.0-released.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Crystal 1.9.0 is released!
+version: 1.9.0
+summary:
+thumbnail: +
+author: straight-shoota
+---
+
+We are delivering a new Crystal release with several bugfixes and improvements.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.9.0)
+and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for
+installation instructions.
+
+## Stats
+
+This release includes [XXX changes since 1.8.2](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.9.0)
+by XX contributors. We thank all the contributors for all the effort put into
+improving the language! ❤️
+
+## Changes
+
+Below we list the most remarkable changes in the language, compiler and stdlib.
+For more details, visit the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.9.0).
+
+### Breaking: Big number comparisons with floats
+
+Comparisons of Big number types against floats were fixed to respect special
+values such as `NAN`. As a result, the implementation of the comparison operator
+`#<=>` between `Big*` (`BigDecimal`, `BigFloat`, `BigInt`, `BigRational`) and
+`Float::Primitive` (`Float32`, `Float64`) number types is now nilable.
+When invoking these comparisons, `Nil` values must be handled explicitly.
+
+### Windows
+
+The compiler and standard library take a big step improving Windows support.
+Only few outstanding issues remain and we expect them to be resolved in the
+following months.
+
+Most notable features are a GUI installer (available in the [release artifacts](https://github.com/crystal-lang/crystal/releases/tag/1.9.0))
+and dynamic linking of DLLs (currently available via compiler flag
+`-Dpreview_dll`).
```

Comment:
Is it worth mentioning [this issue](https://github.com/crystal-lang/crystal/issues/13644) with respect to `-Dpreview_dll`?

Reply:
I don't think this is necessary. The behaviour is fixed (no delay loading by default) and everything else explained in the Windows article.